main/getting-started/Documentation-Map.md

```diff
@@ -0,0 +1,69 @@
+# Agoric Documentation Map
+
+**tyg todo**: Add links to all these docs before publishing
+
+Agoric provides three main types of documentation, as well as links to
+"big picture" presentations and papers and links to our source code on GitHub. This page
+describes our documentation structure, where documents are, and when you should use them, 
+as well as what order to read them.
+
+Our three main documentation types are:
+- **Getting Started**: These are the first things you should
+read. They cover what to do to install, set up, and start
+using Agoric's smart contract tools. They also introduce those tools and
+their underlying concepts and designs.
+- **Guides**: These are more detailed and complete descriptions
+of our tools and their concepts, designs, and usage. They include
+descriptions and examples of how to do various tasks with Agoric
+software.
+- **Reference**: These are the specs for our APIs and commands,
+  as well as checklist tables that specify and explain the
+  steps for extended procedures, such as installing Agoric
+  software.
+
+Some documents fall into multiple categories. For example, our Agoric
+installation document is both a Reference and a Getting Started document.
+
+You should read them in this order:
+1. **Agoric Prerequisites**: What you need to do before installing
+   Agoric software and installing the Agoric SDK.
+2. **Introduction to ERTP**: A Getting Started document that
+  briefly explains the concepts and components of Agoric's Electronic
+  Rights Transfer Protocol (ERTP) which is the base of Agoric
+  software.
+3. **Introduction to Zoe**: A Getting Started document that
+  briefly explains the concepts and components of Zoe, our platform
+  for writing smart contracts in JavaScript. Zoe implements our smart
+  contract guarantee that users either get what they wanted from the
+  contract or a full refund of what they put up for it. Regardless of
+  whether the contract is honest, buggy, or malicious.
+4. **Using An Agoric Demo Contract**: Agoric supplies several
+  demo smart contracts for you to experiment with, learn from, and
+  even copy code from for use in your projects. This document shows
+  you how to install and modify the Autoswap demo contract.
+5. **ERTP Guide**: A detailed description of ERTP concepts, 
+  design, components, and commands. Includes examples of command 
+  usage.
+6. **Zoe Guide**: A detailed description of Zoe concepts, 
```

Comment:
I think if we can describe both guides using the same words, it is too vague. Maybe we can include more ERTP or Zoe specific concepts?